CatAnalyzer/test/TTdilepDraw/TTdilepDraw.py

```diff
@@ -1,21 +1,32 @@
 import ROOT,copy
-import array
-import sys
-
-def hist_maker(name, title, bin_set, x_name, y_name, tr, br, cut):
-    if title == "ll_m":
-        massbin = array.array('d', [20, 30, 45, 75, 105, 125, 165, 260, 400])
-        hist = ROOT.TH1F(name, title, 8, massbin)
-    else: hist = ROOT.TH1F(name, title, bin_set[0], bin_set[1], bin_set[2])
+from array import array
+
+def hist_maker(name, title, bins, x_name, y_name, tr, br, cut):
+    hist = ROOT.TH1F(name, title, len(bins)-1, array('f', bins))
     hist.SetStats(0)
     tr.Project(name, br, cut)
+    if "1" in title:
+        hist2 = ROOT.TH1F(name+"2", title, len(bins)-1, array('f', bins))
+        br = plotvar.replace("1", "2")
+        tr.Project(name+"2", br, cut)
+        hist.Add(hist2)
```

Comment:
Is it safe? What happens if title contains "1" with different meaning - say hLepton1Pt etc.. Wouldn't it better to check suffix only?